### PR TITLE
update pod scheduling readiness gate in pod lifecycle

### PR DESCRIPTION
**Pull request by lianghao208**
Part of https://github.com/kubernetes/kubernetes/issues/113608

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -267,6 +267,38 @@ after successful sandbox creation and network configuration by the runtime
 plugin). For a Pod without init containers, the kubelet sets the `Initialized`
 condition to `True` before sandbox creation and network configuration starts.
 
+### Pod scheduling readiness {#pod-scheduling-readiness-gate}
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md:270]: We have a page `/docs/concepts/scheduling-eviction/pod-scheduling-readiness` dedicated to this topic.
I think we should add a link to that page instead.

Comment (lianghao208) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md:270]: Can you elaborate where do I add the link? Simply add it in this section with a link or elsewhere in this page?

Comment (lianghao208) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md:270]: updated.

**Comment by reylejano** [comment]
Related KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/3521-pod-scheduling-readiness

/approve

**Pull request merged**